### ISSUE TITLE
Flash animation appears in the wrong location

### ISSUE THREAD

**Issue by Mathijssch** [ManimCommunity/manim#2338]
## Description of bug / unexpected behavior
Since version 0.12.0, the `Flash` animation around a Mobject always seems to play in the center of the screen rather than on the provided Mobject.

## How to reproduce the issue
The example [`UsingFlash`](https://docs.manim.community/en/stable/reference/manim.animation.indication.Flash.html#usingflash) in the documentation exhibits this problem. For convenience, I paste the code here:

<details><summary>Code for reproducing the problem</summary>

```py
from manim import *

class UsingFlash(Scene):
    def construct(self):
        dot = Dot(color=YELLOW).shift(DOWN)
        self.add(Tex("Flash the dot below:"), dot)
        self.play(Flash(dot))
        self.wait()
```
</details>

**Comment by behackl** [comment]
Duplicate of #2257, already fixed via #2292 which will be included in the upcoming release.

**Comment by behackl** [comment]
(See the latest version of the documentation for reference: https://docs.manim.community/en/latest/reference/manim.animation.indication.Flash.html :-))
